hudi-common/src/main/java/org/apache/hudi/index/expression/HoodieExpressionIndex.java

```diff
@@ -47,6 +47,10 @@ public interface HoodieExpressionIndex<S, T> extends Serializable {
   String DAYS_OPTION = "days";
   String FORMAT_OPTION = "format";
   String IDENTITY_TRANSFORM = "identity";
+  // Bloom filter options
+  String BLOOM_FILTER_TYPE = "filterType";
+  String BLOOM_FILTER_NUM_ENTRIES = "numEntries";
+  String FALSE_POSITIVE_RATE = "fpp";
```

Comment:
Should we avoid case sensitivity and only use lower case letters in the option names?

Reply:
Don't have a string preference. I just thought it's easy to read with camel case. Do we have a convention? I can change if so.